### PR TITLE
Created Webinar Model

### PR DESCRIPTION
**Pull request by miyurugunarathna**
Completed #18  Create Webinar Model.

### CODE REVIEW

File: src/api/interfaces/IWebinar.ts [hunk 1]
```diff
@@ -1,3 +1,14 @@
-/**
- * @todo implement the @interface IWebinar 
- */
\ No newline at end of file
+import { Document } from "mongoose";
+
+interface IWebinar extends Document {
```
Comment (senuravihanjayadeva) [inline src/api/interfaces/IWebinar.ts:3]: Please use following attribute names when naming.

       "imageUrl": "/assets/events/pastwebinars/ITPMERN.svg",
      "time": "6.00 PM",
      "dateTime": "2021-07-12T18:00:00",
      "title": "ITP GUIDE 2.0 - KICKSTART YOUR MERN APPLICATION WITH GitHub",
      "description": "Practical experience on how to implement your IT project with insight on different tools and technologies",
      "tags": ["mern", "github", "itp", "msclubofsliit", "supportsessions", "growtogether"],
      "link": "https://www.facebook.com/msclubsliit/photos/2678921442232089"

**Review by senuravihanjayadeva — CHANGES_REQUESTED** [review]

File: src/api/interfaces/IWebinar.ts (hunk 1 above)
Comment (miyurugunarathna) [reply src/api/interfaces/IWebinar.ts:3]: Done

File: src/api/interfaces/IWebinar.ts [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+    title: string;
+    description: string;
+    imageUrl: string;
+    dateTime: Date;
+    time: Date;
+    tags?: string[];
+    link?: string;
+    isDeleted: boolean;
```
Comment (senuravihanjayadeva) [inline src/api/interfaces/IWebinar.ts:11]: change isDelete to deletedAt and the data type should be Date.
deletedAt: { type: Date, required: fasle, default: null }
When we get the webinar information just check whether deletedAt field is null or not. null means its not deleted. Please Look at my Pr also.
https://github.com/ms-club-sliit/msclub-backend/pull/28

File: src/api/models/Webinar.model.ts [hunk 3]
```diff
@@ -1,3 +1,20 @@
...
+const WebinarSchema = new Schema<IWebinar>(
+    {
+        title: { type: String, required: true },
+        description: { type: String, required: true },
+        imageUrl: { type: String, required: true },
+        dateTime: { type: Date, required: true },
+        time: { type: Date, required: true },
+        tags: [{ type: String, required: false }],
+        link: { type: String, required: true },
+        isDeleted: { type: Boolean, required: true },
```
Comment (senuravihanjayadeva) [inline src/api/models/Webinar.model.ts:13]: change isDelete to deletedAt and the data type should be Date.
deletedAt: { type: Date, required: fasle, default: null }
When we get the webinar information just check whether deletedAt field is null or not. null means its not deleted. Please Look at my Pr also.
https://github.com/ms-club-sliit/msclub-backend/pull/28

**Review by senuravihanjayadeva — CHANGES_REQUESTED** [review]

File: src/api/interfaces/IWebinar.ts (hunk 2 above)
Comment (miyurugunarathna) [reply src/api/interfaces/IWebinar.ts]: Done

File: src/api/models/Webinar.model.ts (hunk 3 above)
Comment (miyurugunarathna) [reply src/api/models/Webinar.model.ts]: Done

**Review by rusiruavb — APPROVED** [review]
LGTM

**Review by senuravihanjayadeva — APPROVED** [review]

**Pull request merged**